compiler/il/OMRNode.cpp

```diff
@@ -3359,6 +3359,60 @@ OMR::Node::getOwningMethod()
 
 
 
+/** \brief
+ *  Used to get the ram method from the Bytecode Info.
+ * 
+ *  \param *comp
+ *  _compilation of type TR::Compilation 
+ *  
+ *  @return 
+ *  Returns the ram method (TR_OpaqueMethodBlock) from function call getOwningMethod(TR::Compilation *comp, TR_ByteCodeInfo &bcInfo)
+ */
+TR_OpaqueMethodBlock* 
+OMR::Node::getOwningMethod(TR::Compilation *comp)
+   {
+   return OMR::Node::getOwningMethod(comp, self()->getByteCodeInfo());
```

Comment:
Do you mean to specifically call the `OMR` version of this function?  Generally you should call `TR::Node::getOwningMethod` to pick up any specializations of that method that might exist in downstream projects, unless the OMR version is always the one you want.

Reply:
In this case it is `OMR::Node::getOwningMethod`. I'm trying to call the other function I defined here. Please let me know if you'd want me to change anything.

Reply:
Because Node is an extensible class, it is conceivable that `getOwningMethod(TR::Compilation *, TR_ByteCodeInfo &)` might be extended by a downstream project.  To handle that possibility, this code should really be:
```
return TR::Node::getOwningMethod(comp, self()->getByteCodeInfo());
```
unless there is some reason why your implementation **must** call this OMR-specific version.

Anecdotally, we have encountered bugs in the past where the problem turned out to be that the most-specialized version of a function wasn't called.  It's a bit hard to enforce this with extensible classes as it puts the responsibility on the developer for doing the right thing (not ideal, I know).